tests/rados/test_node_drain_customer_bug.py

```diff
@@ -0,0 +1,292 @@
+        if "_no_schedule" in node["labels"]:
+            drain_host = node["hostname"]
+            break
+    if drain_host is None:
+        osd_hosts = rados_obj.get_osd_hosts()
+        log.info(f"The osd nodes list are-{osd_hosts}")
+        drain_host = random.choice(osd_hosts)
+
+    init_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+    log.info(f"The test execution  started at - {init_time} ")
+    try:
+        osd_count_before_test = get_node_osd_list(rados_obj, ceph_nodes, drain_host)
+        log.info(
+            f"The OSDs in the drain node before starting the test - {osd_count_before_test} "
+        )
+        mgr_daemon.start()
+        service_obj.remove_custom_host(host_node_name=drain_host)
+        time.sleep(300)
+        end_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+        log.info(f"The test execution  ends at - {end_time}")
+        if not verify_mgr_traceback_log(
+            rados_obj=rados_obj,
+            start_time=init_time,
+            end_time=end_time,
+            mgr_daemon_list=mgr_daemon_list,
+            mgr_host_object_list=mgr_host_object_list,
+            lines=log_lines,
+        ):
+            log.error("Traceback messages are noticed in logs")
+            return 1
+        log.info(
+            "Adding the node by providing the deploy_osd as False, because the script is not setting the "
+            "--unmanaged=true.Once the node is added back to the cluster the OSDs get configured automatically"
+        )
+        service_obj.add_new_hosts(add_nodes=[drain_host], deploy_osd=False)
+        end_time = datetime.datetime.now() + datetime.timedelta(minutes=5)
+        osd_add_chk_flag = False
+        while datetime.datetime.now() <= end_time:
+            osd_count_after_test = get_node_osd_list(rados_obj, ceph_nodes, drain_host)
+            log.info(
+                f"The OSDs after adding the drain node are- {osd_count_after_test} "
+            )
+            if len(osd_count_after_test) == len(osd_count_before_test):
+                log.info("All the OSDs are added to the node")
+                osd_add_chk_flag = True
+                break
+            time.sleep(30)
+        if not osd_add_chk_flag:
+            log.error("All the OSDs are not added to the node")
+            return 1
+    except CommandFailed as err:
+        log.error(f"Failed with exception: {err.__doc__}")
+        log.exception(err)
+        exceptional_flag = True
+
+        time.sleep(400)
+        end_time, _ = installer.exec_command(cmd="sudo date '+%Y-%m-%d %H:%M:%S'")
+        if verify_mgr_traceback_log(
+            rados_obj=rados_obj,
+            start_time=init_time,
+            end_time=end_time,
+            mgr_daemon_list=mgr_daemon_list,
+            mgr_host_object_list=mgr_host_object_list,
+            lines=log_lines,
+        ):
+            log.error("Traceback messages are not noticed in logs")
+            return 1
```

Comment:
same as above. add print if the error messages were not found in the logs.

Reply:
I have included the message.